app/views/main.scala.html

```diff
@@ -59,6 +59,32 @@
         <div class="govuk-width-container ">
             <div class="govuk-footer__meta">
                 <div class="govuk-footer__meta-item govuk-footer__meta-item--grow">
+                    <ul class="govuk-footer__inline-list">
+                        <li class="govuk-footer__inline-list-item">
+                          <a class="govuk-footer__link" href="@routes.HelpGuideController.helpGuide()">
+                            Help
+                          </a>
+                        </li>
+                        <li class="govuk-footer__inline-list-item">
+                          <a class="govuk-footer__link" href="#">
```

Comment:
Like we said in sprint planning a couple of weeks ago, let's skip any links for now if we don't have the pages yet. It's not hard to add them in later, and it saves confusion by not having a broken link in the meantime.

Reply:
No worries, I'll remove the link for 'contact' for now then :)